src/components/MegaButton/__tests__/MegaButton.spec.js

```diff
@@ -0,0 +1,125 @@
+import '@testing-library/jest-dom';
+import { render, fireEvent } from '@testing-library/vue';
```

Comment:
We should be using `userEvent` (from `import userEvent from '@testing-library/user-event';`) in place of `fireEvent` for most events, especially `click`.  `userEvent` more accurately emulates user behavior and the firing of the events than `fireEvent` does.  `fireEvent.click` has been known to bypass things like a button being disabled and fires the event anyway.